prowler/providers/aws/services/iam/iam_role_cross_account_readonlyaccess_policy/iam_role_cross_account_readonlyaccess_policy.py

```diff
@@ -0,0 +1,70 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.iam.iam_client import iam_client
+
+
+class iam_role_cross_account_readonlyaccess_policy(Check):
+    def execute(self) -> Check_Report_AWS:
+        findings = []
+        for role in iam_client.roles:
+            report = Check_Report_AWS(self.metadata())
+            report.region = iam_client.region
+            report.resource_arn = role.arn
+            report.resource_id = role.name
+            report.resource_tags = role.tags
+            report.status = "PASS"
+            report.status_extended = (
+                f"IAM Role {role.name} has not ReadOnlyAccess policy"
+            )
+            for policy in role.attached_policies:
```

Comment:
What about changing the `role.attached_policies` to a map since if there are thousands of roles this is going to loop for all.

Reply:
A role can have a maximum of 20 attached policies, I don't think will impact the performance.

Reply:
Enough, thanks!